### ISSUE TITLE
[Concept]: Add Non-X (X = specific location) to represent locations that are not X

### ISSUE THREAD

**Issue by coolharsh55** [w3c/dpv#208]
### Specs

Locations (LOC)

### New Concept(s)

The LOC extension should also provide concepts that represent something is _not this location_ e.g. `loc:NonEU` means all locations that are not EU (or in EU), and `loc:NonIE` means all locations that are not or in Ireland (IE). This will assist implementations where they have to specify locations in the context of rules or conditions, such as no data transfers outside of EU. 

Without the additional concept, this requires a logical operator (_NOT_) and a location concept (`loc:EU`) - which introduces complexity in representing, sharing, and interpreting the information across use-cases. Instead, if we provided `loc:NonEU` as a concept, it can be directly represented e.g. as `dpv:hasLocation loc:NonEU` within a rule or a use-case - thereby simplifying the way such conditions or situations are represented.

The proposal is to create and provide such NonX concepts for all countries and supraunions (EU, EEA, etc.) as these would at least be the most commonly used concepts.

**Comment by bact** [comment]
```ttl
loc:NonEU a dpv:SupraNationalUnion ;
  a loc:InvertedLocation ;
  loc:isInvertedLocationOf loc:EU .
```

Is `a dpv:SupraNationalUnion` necessary here? (or even valid, as while EU is a supranation but non-EU is not a supranation)

**Comment by coolharsh55** [comment]
@bact you're correct, it shouldn't be a supranation (the mistake was because I took definition of `loc:EU` to start with.

**Comment by coolharsh55** [comment]
(using [dpvbot](https://github.com/coolharsh55/dpv-x/tree/master/dpvbot):) This was discussed in [Meeting 2025-02-13](https://w3id.org/dpv/meetings/meeting-2025-02-13.html) 
We should start with EU/nonEU concepts first, and then later add other country or region specific nonX concepts on a case-by-case basis to ensure they are correct and useful.

**Comment by coolharsh55** [comment]
(using [dpvbot](https://github.com/coolharsh55/dpv-x/tree/master/dpvbot):) This was discussed in [Meeting 2025-04-17](https://w3id.org/dpv/meetings/meeting-2025-04-17.html) 
Accepted to add these to DPV 2.2

**Comment by bact** [comment]
For subdivisions in ISO 3166-2 that are also included in ISO 3166-1 ("country" codes), see:
https://en.wikipedia.org/wiki/ISO_3166-2#Subdivisions_included_in_ISO_3166-1

**Comment by coolharsh55** [comment]
fantastic, thanks @bact -- I'll see how to incorporate that. Easiest way I think is to change the type of those locations to `dpv:Region` instead of `dpv:Country` with the listed country as parent/broader, and then this also removes them from the `non-X` sets. Though for Taiwan/TW this means it gets listed under China/CN as per ISO and UN taxonomies -- maybe we should add a note regarding this alongside the (todo) note regarding geo-political implications that we discussed in the last meeting?

**Comment by coolharsh55** [comment]
4e16f30d86b3ef991bd5929d781c26a1ccf998c6 fixes the above concepts by declaring them as `dpv:Region` instead of `dpv:Country` and with the associated country as `skos:broader`. They also no longer have inverse jurisdiction sets defined, or participate in any such sets (as they are regions, and these sets are only for countries). 

This also made me wonder whether we should declare or annotate some of these concepts as _disputed_ or other label to indicate they are politically contested or not uniformly acknowledged across jurisdictions (which can be the trigger to create & use a jurisdiction-specific taxonomy).
